src/supertux/menu/main_menu.cpp

```diff
@@ -89,7 +90,7 @@ MainMenu::menu_action(MenuItem* item)
       break;
 
     case MNID_QUITMAINMENU:
-      if (true)
+      if(!g_config->confirmation_dialog)
```

Comment:
Keep the space in between if and the parenthesis.

Also this is really silly and I don't think this should actually happen, we should rather throw out the confirmation dialog here. If a user wants to quit SuperTux, they really want to. And it's not like progress is lost when quitting properly from the menu.

Reply:
should i remove it?